child_compassion/models/project_lifecycle_event.py

```diff
@@ -56,8 +69,65 @@ class ProjectLifecycle(models.Model):
 
     # Transition
     ############
-    actual_transition_date = fields.Date(readonly=True)
+    date = fields.Date(readonly=True)
```

Comment:
rename to `transition_date`
the `date` field will be used simply to know when we received the information